tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -185,6 +185,31 @@ describe(__filename, () => {
     expect(link).toHaveProp('href', 'http://license.com/');
   });
 
+  it('renders the license info without a link if the url is null', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        license: { name: 'justText', url: null },
+      },
+    });
+    const root = render({ addon });
+    const link = root.find('.AddonMoreInfo-license-type');
+    expect(link.children()).toIncludeText('justText');
+  });
+
+  it('does not render any license info if the license is null', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        license: null,
+      },
+    });
+    const root = render({ addon });
+    expect(root.find('.AddonMoreInfo-license')).toHaveLength(0);
+  });
```

Comment:
I am surprised that this test case did not exist before.

Reply:
yeh me too..